### PR TITLE
community items: accessibility & responsiveness improvements

### PR DESCRIPTION
**Pull request by jennur**
Closes https://github.com/inveniosoftware/invenio-app-rdm/issues/2182

**Needs** https://github.com/inveniosoftware/react-invenio-forms/pull/199
- Accessibility fixes
- Adds popup for review information (optional through `showPermissionLabel`-prop)
Screenshot 2023-04-28 at 16 14 28

### CODE REVIEW

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js [hunk 1]
```diff
@@ -4,41 +4,40 @@
...
+  const { metadata, ui, links, access, id } = result;
   const communityType = ui?.type?.title_l10n;
 
   return (
     <Item
       key={id}
       className={`computer tablet only justify-space-between community-item ${itemClassName}`}
     >
-      <Image as={Item.Image} size="tiny" src={links.logo} />
+      <Image size="tiny" src={links.logo} alt="" />
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js:31]:
```suggestion
      <Image size="tiny" src={links.logo} alt={i18next.t("Community logo")} />
```

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js [hunk 2]
```diff
@@ -55,22 +54,56 @@ export const CommunityCompactItemComputer = ({
...
+            <Item.Content>
+              <Item.Meta>
+                {ui?.permissions?.can_include_directly && (
+                  <InvenioPopup
+                    popupId="direct-publish-info-popup"
+                    size="small"
+                    trigger={<Icon name="paper plane outline" size="large" />}
+                    ariaLabel={i18next.t("Submission information")}
+                    content={i18next.t(
+                      "Submission does not require review, and will be published directly."
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js:68]:
```suggestion
                      "Submission does not require review, and will be published immediately."
```

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js [hunk 3]
```diff
@@ -55,22 +54,56 @@ export const CommunityCompactItemComputer = ({
...
+                    popupId="requires-review-popup"
+                    size="small"
+                    ariaLabel={i18next.t("Submission information")}
+                    trigger={
+                      <span>
+                        <Icon name="comments outline" size="large" />
+                        <Icon corner="top right" name="question" size="small" fitted />
+                      </span>
+                    }
+                    content={i18next.t("Submission requires review.")}
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js:83]:
```suggestion
                    content={i18next.t("Submission to this community requires review and will be published upon curator's approval.")}
```

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemMobile.js [hunk 4]
```diff
@@ -17,13 +17,15 @@ export const CommunityCompactItemMobile = ({ result, actions }) => {
 
   return (
     <Item key={result.id} className="mobile only justify-space-between">
-      <Image size="mini" src={result.links.logo} />
+      <Image size="mini" src={result.links.logo} alt="" />
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemMobile.js:20]:
```suggestion
      <Image size="mini" src={result.links.logo} alt={i18next.t("Community logo")} />
```

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityItemComputer.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ export const CommunityItemComputer = ({ result }) => {
         wrapped
         src={result.links.logo}
         className="community-logo"
+        alt=""
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityItemComputer.js:26]:
```suggestion
        alt={i18next.t("Community logo")}
```

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityItemMobile.js [hunk 6]
```diff
@@ -24,7 +24,7 @@ export const CommunityItemMobile = ({ result, index }) => {
           <CommunityTypeLabel type={communityType} />
         </Item.Extra>
         <Item.Extra>
-          <Image wrapped src={result.links.logo} size="small" />
+          <Image wrapped src={result.links.logo} size="small" alt="" />
```
Comment (kpsherva) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityItemMobile.js:27]:
```suggestion
          <Image wrapped src={result.links.logo} size="small" alt={i18next.t("Community logo")} />
```

File: invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js (hunk 1 above)
Comment (jennur) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesItems/CommunityCompactItemComputer.js:31]: I don't think this is needed, as the logo doesn't have any functionality and doesn't provide any information, ref [https://www.w3.org/WAI/tutorials/images/](https://www.w3.org/WAI/tutorials/images/)

**Review by kpsherva — APPROVED** [review]

**Pull request merged**